index.html

```diff
@@ -74,49 +72,66 @@ <h2 class="visually-hidden">Достопримечательности</h2>
         </li>
       </ul>
     </section>
-
-    <!--здесь окончание разметки доп задания-->
     <section class="search-hotel">
       <h2 class="visually-hidden">Поиск отеля</h2>
       <p>Заинтересовались?</p>
       <p>Укажите предполагаемые даты поездки, и мы покажем Вам лучшие предложения гостиниц в Седоне</p>
-      <p><a class="button" href="search-form.html">Поиск гостиницы в Седоне</a></p>
+      <a class="button" href="search-form.html">Поиск гостиницы в Седоне</a>
     </section>
     <section class="modal modal-search-form">
       <h2 class="visually-hidden">Форма поиска отеля</h2>
-      <!--Здесь будет форма-->
+      <form class="filter" action="https://echo.htmlacademy.ru" method="get">
+        <label class="date-check-in">
+          Дата заезда:<input type="text" required name="date-check-in" value="24 апреля 2017">
+          <!--Если тут поставить type date то значение по умолчанию затрется на ДД.ММ.ГГГГ
+        значок календарика будем ставить в CSS как background потом?-->
+        </label>
+        <label class="date-check-out">
+          Дата выезда:<input type="text" required name="date-check-out" value="4 июля 2017">
+        </label>
+        <button type="button" name="minus">-</button>
```

Comment:
У тебя получается по верстке будет идти кнопка минус потом ВЗРОСЛЫЕ потом счетчик потом кнопка плюс

Нужен такой порядок
Лэйбл, кнопка, счетчик, кнопка.

То есть придется вынести из лэйбла инпут и связять их через атрибут for

```suggestion
        <label for="adult">Взрослые:</label>
        <input type="text" id="adult" name="adult" value="2">
```
Иначе потом не сверстать по макету. Плюс ко всему тебе надо будет врапать это во что то, чтобы сделать 2 колонки. Но это другая история.